scapegoat.ui/src/App.js

```diff
@@ -22,13 +22,12 @@ function App() {
       if (authed) {
         authed.getIdToken()
         .then((token) => sessionStorage.setItem('token', token));
-        // get each thing
         setUser(user);
```

Comment:
I think we need to pass `authed` to `setUser` instead of `user`

Reply:
thank you!